kube/resources/webapp.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: airbyte-webapp-svc
+spec:
+  type: NodePort
+  ports:
+    - port: 80
+      protocol: TCP
+  selector:
+    airbyte: webapp
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airbyte-webapp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      airbyte: webapp
+  template:
+    metadata:
+      labels:
+        airbyte: webapp
+    spec:
+      containers:
+        - name: airbyte-webapp-container
+          image: airbyte/webapp
+          env:
+            - name: AIRBYTE_VERSION
+              value: 0.7.2-alpha # todo
```

Comment:
to do?

Reply:
again, anyway we can pull these values from .env?

Reply:
the more i look at this the more this is my biggest concern. the current case where we have what used to be 1 .env file spread across a ton of files is a little scary. if we need 2 .env files (one for docker one for k8s) that seems tolerable. but right now with the env variables spread everywhere, i think we are going to have problems.

Reply:
the way to do it is to use https://kubernetes.io/docs/concepts/configuration/configmap/

This way we can have only one place where we store all these parameters